packages/@aws-cdk/aws-apigatewayv2/lib/common/integration.ts

```diff
@@ -9,4 +9,36 @@ export interface IIntegration extends IResource {
    * @attribute
    */
   readonly integrationId: string;
-}
\ No newline at end of file
+}
+
+/**
+ * AWS service integration sub types
+ * https://docs.aws.amazon.com/apigateway/latest/developerguide/http-api-develop-integrations-aws-services-reference.html
+ */
+export enum AwsServiceIntegrationSubtype {
+  EVENT_BRIDGE_PUT_EVENTS = 'EventBridge-PutEvents'
+  // AppConfig-GetConfiguration
+  // Kinesis-PutRecord
+  // SQS-DeleteMessage
+  // SQS-PurgeQueue
+  // SQS-ReceiveMessage
+  // SQS-SendMessage
+  // StepFunctions-StartExecution
+  // StepFunctions-StartSyncExecution
+  // StepFunctions-StopExecution
+}
+
+/**
+ * Integration request parameters for EventBridge
+ */
+export interface EventBridgeIntegrationRequestParameters {
+  readonly detail?: string;
+  readonly detailType?: string;
+  readonly eventBusName?: string;
+  readonly region?: string;
+  readonly resources?: string[];
+  readonly source?: string;
+  readonly time?: string;
+}
+
+export type AwsServiceIntegrationRequestParameters = EventBridgeIntegrationRequestParameters;
```

Comment:
Unfortunately, type aliasing is not supported by jsii

Reply:
Bummer, unions then?

Reply:
See below.

Reply:
neither unions, unfortunately.